api/core/agent/cot_completion_agent_runner.py

```diff
@@ -19,11 +19,11 @@ def _organize_instruction_prompt(self) -> str:
         
         return system_prompt
 
-    def _organize_historic_prompt(self) -> str:
+    def _organize_historic_prompt(self, prompt_message: list[PromptMessage] = []) -> str:
```

Comment:
It's not recommended to use a default value which has a mutable type, it brings lots of problems like [this](https://stackoverflow.com/questions/1132941/the-mutable-default-argument-in-python), I prefer setting it to None and assign an empty array to it if it's None at the beginning of this method.

Reply:
optimized